src/components/Checkout/OrderTotals.vue

```diff
@@ -391,11 +392,11 @@ export default {
 				price: numeral(this.totals.donationTotal)
 			};
 		},
-		isCorporateCampaignPage() {
-			return this.$route.path.substring(0, 4) === '/cc/';
-		}
 	},
 	methods: {
+		ccPageCheck(route) {
+			return isCCPage(route);
```

Comment:
Usage could be simplified by instead using `this.$route` in this method instead. And it could then be converted into a computed method.

```
ccPageCheck() {
			return isCCPage(this.$route);
}
```

Reply:
I agree, the existing computed prop can just be altered to the following so names across the file don't need to be changed.

```
isCorporateCampaignPage() {
    return isCCPage(this.$route);
}
```